samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +349,35 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	int64_t time_now;
+	char buf[SAMPLE_MSG_BUF_SIZE];
+
+	/* Get the current timestamp */
+	err = date_time_now(&time_now);
+	if (err) {
+		LOG_ERR("Failed to get timestamp for Hello World message");
+		return err;
```

Comment:
Perhaps still send the message, but have the timestamp be `-1`? This would require checking for `-1` inside our internal tests, though. Up to you.

Reply:
When this fails, `time_now` will be some random number as `date_time_now` will retain its old value.

Reply:
Actually, upon re-reading, I see that you're adding the timestamp as a key rather than as text in the message. So don't bother with sending it when a timestamp is not available, it isn't worth the added complexity in the code.

That being said, I think you should add it both as a ts key, and as text in the message

Reply:
i too am confused